### PR TITLE
make integer64 available via re-export (closes #15)

### LINKED ISSUE #15: as.integer64 returns an error
I'm not sure if this is exactly an _issue_ with `nanotime` but since `as.integer64.nanotime` is listed as a function in the help file for it, I wasn't expecting this behavior.

``` r
library(nanotime)
nt <- nanotime("2017-02-02T17:16:27.633531000+00:00")
as.integer64(nt)
#> Error in eval(expr, envir, enclos): could not find function "as.integer64"
suppressMessages(library(bit64))
as.integer64(nt)
#> integer64
#> [1] 1486055787633531000
```

Should I have to explicitly load `bit64` for this to work or is there a configuration issue with `nanotime`? Perhaps this is just a lack of understanding on my part.

### PR DESCRIPTION
**Pull request by eddelbuettel**

### CODE REVIEW

File: man/nanotime.Rd [hunk 1]
```diff
@@ -85,6 +88,9 @@ A nanotime object
 Functions to operate on nanosecond time resolution using integer64
 bit representation. Convertions functions for several standard R
```
Comment (restonslacker) [inline man/nanotime.Rd]: 'Convertions' -> 'Conversion'

Comment (restonslacker) [inline man/nanotime.Rd]: also, further down:

'64 bit integers whereas R itself _onky_ has 32 bit integers' ->
'64-bit integers whereas R itself _only_ has 32-bit integers'

don't know if you want 32- and 64-bit hyphenated.  you use both in this document

**Comment by restonslacker** [comment]
This appears to work as i would expect it to. Thanks Dirk!

File: man/nanotime.Rd (hunk 1 above)
Comment (eddelbuettel) [inline man/nanotime.Rd]: Good catches, both typos look old.  I should check the homepage.

Also made hyphenation more consistent.

**Pull request merged**